pkg/apis/externaldns/types.go

```diff
@@ -122,7 +124,8 @@ func (cfg *Config) ParseFlags(args []string) error {
 	// Miscellaneous flags
 	app.Flag("log-format", "The format in which log messages are printed (default: text, options: text, json)").Default(defaultConfig.LogFormat).EnumVar(&cfg.LogFormat, "text", "json")
 	app.Flag("metrics-address", "Specify where to serve the metrics and health check endpoint (default: :7979)").Default(defaultConfig.MetricsAddress).StringVar(&cfg.MetricsAddress)
-	app.Flag("debug", "When enabled, increases the logging output for debugging purposes (default: disabled)").BoolVar(&cfg.Debug)
+	app.Flag("debug", "Shortcut for --log-level=debug. When enabled, increases the logging output for debugging purposes (default: disabled)").BoolVar(&cfg.Debug)
+	app.Flag("log-level", "Set the level of logging. (default: info, options: debug, info, warn, error, fatal").Default("info").StringVar(&cfg.LogLevel)
```

Comment:
Please use `EnumVar` here (see `log-format` flag above for example). This will let you drop some of the validation code.

For the `Default()` please use `defaultConfig.LogLevel`.

Reply:
Done.